### DISCUSSION TITLE
Is it possible to export the Metrics as csv/Excel?

### DISCUSSION THREAD

**Discussion by pawarbi** [DaxStudio/DaxStudio#988] [Q&A] [answered]
Hi Darren/Marco - once I generate the Vertipaq metrics with " View Metrics", is it possible to export the metrics as a csv/Excel instead of vpax? I don't need all the tabs in the vpax or use filtering etc, just need metrics results in tabular fashion for data validation purposes. I don't think it's possible but thought of asking. Thanks.

**Comment by marcosqlbi** [comment, marked as answer]
You could copy/paste the content of a table in View Metrics to Excel, but the best way to achieve your goal is to export the VPAX file and import it in Excel using VertiPaq Analyzer: https://www.sqlbi.com/tools/vertipaq-analyzer/

**Reply by pawarbi** [reply]
Thanks, copy/paste worked. See you in couple of days at Chicago PBUG!
